browser/html/cool.html.m4

```diff
@@ -201,21 +201,22 @@ m4_ifelse(MOBILEAPP,[true],
       </div>
      </nav>
 
-     <table id="toolbar-wrapper">
-     <tr>
-       <td id="toolbar-logo"></td>
-       <td id="toolbar-mobile-back" class="editmode-off"></td>
-       <td id="toolbar-up"></td>
-       <td id="toolbar-hamburger">
-         <label class="main-menu-btn" for="main-menu-state">
-           <span class="main-menu-btn-icon" id="main-menu-btn-icon"></span>
-         </label>
-       </td>
-     </tr>
-     <tr>
-       <td colspan="4" id="formulabar" style="display: none"></td>
-     </tr>
-    </table>
+     <div id="toolbar-wrapper">
+        <div id="toolbar-row" class="toolbar-row">
+          <div id="toolbar-logo"></div>
+          <div id="toolbar-mobile-back" class="editmode-off"></div>
+          <div id="toolbar-mobile-center-space">
+            <div id="PermissionMode" class="cool-font status-readonly-mode" style="padding: 3px 10px; display: none"> Read-only</div>
```

Comment:
please remove this `PermissionMode` thing, it should be only generated by JSDialogs builder